samples/boards/nrf52/nffs/README.rst

```diff
@@ -1,11 +1,30 @@
-To successfully run this demo example,
 
-we have to include ext/fs/nffs/include directory
+NFFS: READ WRITE FILE_STATUS
+###########################
 
-into the list of include directories by editing $(zephyr_base)/CMakeLists.txt
+Overview
+********
 
+This is a simple application demonstrating how to read, write persistence data on 
 
-##################### For Example ##########################
+flash of nrf52 SoC using NFFS file system.
+
+
+Requirements
+************
+
+This sample has been tested on the Nordic nRF52840-PDK board, but would
+
+likely also run on the nrf52_pca10040 board.
+
+Building and Running
+********************
+
+To build this example, we have to add "ext/fs/nffs/include" into list of
```

Comment:
How about this instead:

    To build this example, edit the `$ZEPHYR_BASE/CMakeLists.txt`` file to add ``ext/fs/nffs/include``
    to the include directories listed there, so it will look like this:

    .. code-block:: none

      zephyr_include_directories(
        ${STDINCLUDE}
        ext/fs/nffs/include
      )

    After this modification you can build the sample using these commands:

    .. code-block:: bash

       cd $(ZEPHYR_BASE)/samples/boards/nrf52/nffs
       mkdir build
       cd build
       cmake -DBOARD=nrf52840_pca10056 ..
       make
       cd zephyr
       make clean
       make all
 
    After the build completes, the resulting binary ``zephyr.hex`` will be
    in the current directory.  Use this file to flash the board using the
    ``nrfjprog`` utility.

    After flashing to the nrf52840_pdk board, open a serial terminal to see console messages
    written by the application:

    - Press Button 2 on the PDK board to create and write data in file ``0.txt``
    - Press Button 1 to read data from ``0.txt``

    For more information about the Zephyr file system, see the :ref:`file_system` documentation. 

I'm not sure about the build instructions you've written here (I've just made them formatted nicely).  I'd recommend one of the development engineers (@SebastianBoe perhaps) review these build instructions, including how to get the include added (I think you're supposed to create a CMakeLists.txt local to the sample rather than editing the one in ZEPHYR_ROOT.)